alternative-log-storage/aws/dynamodb/README.md

```diff
@@ -0,0 +1,9 @@
+## DynamoDB table on your own cloud provider
+
+You can apply this module on your AWS account, to create a DynamoDB table for storing logs. env0 will assume the role created in this module, in order to read and write logs
+
+#### Variables:
+
+- region - AWS region in which the dynamodb will be created. Should be provided to env0 as well
+- external_id - An external ID provided by env0 for assuming the roles to read and write from and to log table
```

Comment:
Maybe add that info to our wiki - what do we need to provide to the customer. agentKey, externalId

Reply:
Already updated [this wiki](https://github.com/env0/env0/wiki/Migrating-customers-to-K8S-agent), I'll link the actual module once it's merged to master